### PR TITLE
[doc] Add ARGMAX op.

### PR DESCRIPTION
**Pull request by BruceDai**
@huningxin @fujunwei These two new ops were referring to [ARGMAX](https://developer.android.com/ndk/reference/group/neural-networks.html#group___neural_networks_1ggaabbe492c60331b13038e39d4207940e0af78048ff71a1e79bbc6034d30cad7cbc) and [ARGMIN](https://developer.android.com/ndk/reference/group/neural-networks.html#group___neural_networks_1ggaabbe492c60331b13038e39d4207940e0a89be86ff36e83254e9b3d0954e7e0729) of NN API, please take a review, thanks

### CODE REVIEW

File: docs/api.md [hunk 1]
```diff
@@ -62,6 +62,8 @@ interface NeuralNetworkContext {
   const long MAXIMUM = 65;
   const long ATROUS_CONV_2D = 10003;
   const long ATROUS_DEPTHWISE_CONV_2D = 10004;
+  const long ARGMAX = 10005;
+  const long ARGMIN = 10006;
```
Comment (huningxin) [inline docs/api.md]: Could you please align the constant value to NNAPI? 
```
const long ARGMAX = 39;
const long ARGMIN = 40;
```

File: src/nn/Enums.js [hunk 2]
```diff
@@ -1405,6 +1405,42 @@ export const OperationCode = {
...
+   * Supported tensor rank: from 1
+   *
+   * Inputs:
+   * * 0: An n-D tensor specifying the input. Must be non-empty.
+   * * 1: An INT32 scalar specifying the axis to reduce across. Negative index is used to specify axis from the end (e.g. -1 for the last axis). Must be in the range [-n, n).
+   *
+   * Outputs:
+   * * 0: An (n - 1)-D TENSOR_INT32 tensor.
+   */
+   ARGMAX: 10005,
```
Comment (huningxin) [inline src/nn/Enums.js]: ditto

File: src/nn/Enums.js [hunk 3]
```diff
@@ -1405,6 +1405,42 @@ export const OperationCode = {
...
+   * Supported tensor rank: from 1
+   *
+   * Inputs:
+   * * 0: An n-D tensor specifying the input. Must be non-empty.
+   * * 1: An INT32 scalar specifying the axis to reduce across. Negative index is used to specify axis from the end (e.g. -1 for the last axis). Must be in the range [-n, n).
+   *
+   * Outputs:
+   * * 0: An (n - 1)-D TENSOR_INT32 tensor.
+   */
+  ARGMIN: 10006,
```
Comment (huningxin) [inline src/nn/Enums.js]: ditto

File: docs/api.md (hunk 1 above)
Comment (BruceDai) [reply docs/api.md]: Thanks, updated them by the new patch.

**Comment by BruceDai** [comment]
ONNX also supports [ARGMAX](https://github.com/onnx/onnx/blob/master/docs/Operators.md#ArgMax) and [ARGMIN](https://github.com/onnx/onnx/blob/master/docs/Operators.md#argmin) .

**Comment by huningxin** [comment]
> ONNX also supports [ARGMAX](https://github.com/onnx/onnx/blob/master/docs/Operators.md#ArgMax) and [ARGMIN](https://github.com/onnx/onnx/blob/master/docs/Operators.md#argmin) .

Good pointers. Could you please open an issue and document the difference between NNAPI and ONNX for argmax and argmin? Thanks.

**Comment by huningxin** [comment]
Looks good to me. Thanks much!

**Pull request merged**